bin/adhoc-scripts/adjustMongoDocs.py

```diff
@@ -40,34 +47,80 @@ def __init__(self):
                                  dest="dbname", default="", help="MongoDB database")
         self.parser.add_argument("--dbcoll", action="store",
                                  dest="dbcoll", default="", help="MongoDB collection")
+        self.parser.add_argument("--userdn", action="store",
+                                 dest="userDN", default=defaultDN, help="user DN")
+        self.parser.add_argument("--add-tran-record", action="store_true",
+                                 dest="transition", help="add transaction record")
         self.parser.add_argument("--verbose", action="store",
                                  dest="verbose", default=0, help="verbosity level")
 
 
-def adjust():
+def addTransitionRecord(dburi, dbname, dbcoll, userDN, verbose):
+    """
+    Helper function to add and update in place transition record for existing MSPileup records
+    :param dburi: database URI (string)
+    :param dbname: database name (string)
+    :param dbcoll: database collection name (string)
+    :param userDN: string representing user DN
+    :param verbose: verbose flag (boolean)
+    :return: nothing
+    """
+    userDN = '/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=cmsunified/CN=658085/CN=Robot: CMS Unified account'
```

Comment:
I think we need to delete this line and rely on the `defaultDN`, defined in the `__init__` method.

Reply:
ok, done.